gstvstaudioprocessor.cpp

```diff
@@ -1126,12 +1127,15 @@ create_type_name(const gchar * parent_name, const gchar * class_name)
     }
   }
 
-  return typified_name;
+  res = std::string (typified_name);
+  g_free (typified_name);
```

Comment:
Don't do `function (` in C++ code (the space between function name and parenthesis), it looks ugly once you chain method calls :)